x-pack/plugins/apm/ftr_e2e/cypress/tasks/es_archiver.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const esArchiverLoad = (folder: string) => {
```

Comment:
Why not just use the JS API for ESArchiver? https://github.com/elastic/kibana/blob/master/packages/kbn-es-archiver/src/es_archiver.ts

Reply:
@tylersmalley I'm going to leave it the way it is now actually when cypress open and I load the esArchiver, cypress throws an exception.
```
Error: Webpack Compilation Error
/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/build/collectHandles.js
Module not found: Error: Can't resolve 'async_hooks' in '/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/build'
resolve 'async_hooks' in '/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/build'
  Parsed request is a module
  using description file: /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/package.json (relative path: ./build)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/build/node_modules doesn't exist or is not a directory
      /Users/cauemarcondes/dev/kibana/node_modules/@jest/node_modules doesn't exist or is not a directory
      /Users/cauemarcondes/dev/kibana/node_modules/node_modules doesn't exist or is not a directory
      /Users/cauemarcondes/dev/node_modules doesn't exist or is not a directory
      /Users/cauemarcondes/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules
        using description file: /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
      looking for modules in /Users/cauemarcondes/dev/kibana/node_modules
        using description file: /Users/cauemarcondes/dev/kibana/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/package.json (relative path: ./node_modules/async_hooks)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
          using description file: /Users/cauemarcondes/dev/kibana/package.json (relative path: ./node_modules/async_hooks)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.json doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks.json doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.jsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks.jsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.mjs doesn't exist
            .coffee
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks.mjs doesn't exist
            .coffee
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.coffee doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks.coffee doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.tsx doesn't exist
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks.tsx doesn't exist
            as directory
              /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks doesn't exist
            as directory
              /Users/cauemarcondes/dev/kibana/node_modules/async_hooks doesn't exist
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/build/node_modules]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/node_modules]
[/Users/cauemarcondes/dev/kibana/node_modules/node_modules]
[/Users/cauemarcondes/dev/node_modules]
[/Users/cauemarcondes/node_modules]
[/Users/node_modules]
[/node_modules]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks]
[/Users/cauemarcondes/dev/kibana/node_modules/async_hooks]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.js]
[/Users/cauemarcondes/dev/kibana/node_modules/async_hooks.js]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.json]
[/Users/cauemarcondes/dev/kibana/node_modules/async_hooks.json]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.jsx]
[/Users/cauemarcondes/dev/kibana/node_modules/async_hooks.jsx]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.mjs]
[/Users/cauemarcondes/dev/kibana/node_modules/async_hooks.mjs]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.coffee]
[/Users/cauemarcondes/dev/kibana/node_modules/async_hooks.coffee]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.ts]
[/Users/cauemarcondes/dev/kibana/node_modules/async_hooks.ts]
[/Users/cauemarcondes/dev/kibana/node_modules/@jest/core/node_modules/async_hooks.tsx]
[/Users/cauemarcondes/dev/kibana/node_modules/async_hooks.tsx]
 @ /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/build/collectHandles.js 10:39-61
 @ /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/build/cli/index.js
 @ /Users/cauemarcondes/dev/kibana/node_modules/@jest/core/build/jest.js
 @ /Users/cauemarcondes/dev/kibana/node_modules/jest/build/jest.js
 @ /Users/cauemarcondes/dev/kibana/packages/kbn-test/target/jest/run.js
 @ /Users/cauemarcondes/dev/kibana/packages/kbn-test/target/index.js
 @ ./cypress/tasks/es_archiver.ts
 @ ./cypress/support/commands.ts
 @ ./cypress/support/index.ts
```